docs/guides/redis/scaling/horizontal-scaling/sentinel.md

```diff
@@ -0,0 +1,332 @@
+master_replid2:0000000000000000000000000000000000000000
+master_repl_offset:35492
+second_repl_offset:-1
+repl_backlog_active:1
+repl_backlog_size:1048576
+repl_backlog_first_byte_offset:1
+repl_backlog_histlen:35492
+```
+
+Additionally, the sentinel monitoring can be checked with following command : 
+```bash
+kubectl exec -it -n demo sen-sample-0 -c redissentinel -- redis-cli -p 26379 sentinel masters
+```
+
+We are now ready to apply the `RedisSentinelOpsRequest` CR to horizontal scale on sentinel and `RedisOpsRequest` CR to horizontal scale database.
+
+### Horizontal Scale RedisSentinel
+
+Here, we are going to scale down the replicas count of the sentinel to meet the desired resources after scaling.
+
+#### Create RedisSentinelOpsRequest:
+
+In order to scale the replicas of the sentinel, we have to create a `RedisSentinelOpsRequest` CR with our desired number of replicas. Below is the YAML of the `RedisSentinelOpsRequest` CR that we are going to create,
+
+```yaml
+apiVersion: ops.kubedb.com/v1alpha1
+kind: RedisSentinelOpsRequest
+metadata:
+  name: sen-ops-horizontal
+  namespace: demo
+spec:
+  type: HorizontalScaling
+  databaseRef:
+    name: sen-sample
+  horizontalScaling:
+    replicas: 3
+```
+
+Here,
+
+- `spec.databaseRef.name` specifies that we are performing operation on `sen-sample` RedisSentinel instance.
+- `spec.type` specifies that we are going to perform `HorizontalScaling` on our database.
+- `spec.horizontalScaling.replicas` specifies the desired number of replicas after scaling.
+
+Let's create the `RedisSentinelOpsRequest` CR we have shown above,
+
+```bash
+$ kubectl apply -f https://github.com/kubedb/docs/raw/{{< param "info.version" >}}/docs/examples/redis/scaling/horizontal-scaling/horizontal-sentinel.yaml
+redissentinelopsrequest.ops.kubedb.com/sen-ops-horizontal created
+```
+
+#### Verify RedisSentinel replicas updated successfully :
+
+If everything goes well, `KubeDB` Enterprise operator will scale down the replicas of `RedisSentinel` object.
+
+Let's wait for `RedisSentinelOpsRequest` to be `Successful`.  Run the following command to watch `RedisSentinelOpsRequest` CR,
+
+```bash
+$ watch kubectl get redissentinelopsrequest -n demo
+Every 2.0s: kubectl get redissentinelopsrequest -n demo
+NAME                 TYPE              STATUS       AGE
+sen-ops-horizontal   HorizontalScaling   Successful   5m27s
+```
+
+We can see from the above output that the `RedisSentinelOpsRequest` has succeeded.
+
+Let's check the number of master and replicas this database has from the RedisSentinel object
+
+```bash
+$ kubectl get redissentinel -n demo sen-sample -o json | jq '.spec.replicas'
+3
+```
+
+The above output verifies that we have successfully scaled up the resources of the sentinel instance.
+### Horizontal Scale Redis
+
+Here, we are going to update the resources of the redis database to meet the desired resources after scaling.
+
+#### Create RedisOpsRequest:
```

Comment:
HorizontalScaling of Redis has already been discussed in `cluster` page.   As It doesn't matter whether that Redis is referred by a Sentinel or not , We can say that(or refer to) in Cluster page. 

Remove repeatitive parts